script.js

```diff
@@ -1 +1,123 @@
+const form = document.querySelector(".input-box"),
+    input=form.querySelector("input"),
+    waitList=document.querySelector(".wait-list"),
+    completeList=document.querySelector(".complete-list");
 
+const TODO_LS='toDos';
+
+
+
+function loadToDos(){
+    const loadedToDos=localStorage.getItem(TODO_LS);
+    if(loadedToDos!==null){
```

Comment:
```suggestion
    if(loadedToDos) {
```

javascript에서는 truthy한 값을 자동으로 boolean으로 바꿔줘요!
이런 식으로만 해도 돌아간답니당